### PR TITLE
Enables Blobstorage Binding certification test & updates test due to library changes

### PR DESCRIPTION
**Pull request by berndverst**
Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

# Description

All this time the Blob Storage Binding certification was not actually enabled... whoops. This enables the test.

### CODE REVIEW

File: tests/certification/bindings/azure/blobstorage/blobstorage_test.go [hunk 1]
```diff
@@ -558,16 +558,16 @@ func TestBlobStorage(t *testing.T) {
 
 		cred, _ := azblob.NewSharedKeyCredential(os.Getenv("AzureBlobStorageAccount"), os.Getenv("AzureBlobStorageAccessKey"))
 		service, _ := azblob.NewServiceClientWithSharedKey(fmt.Sprintf("https://%s.blob.core.windows.net/", os.Getenv("AzureBlobStorageAccount")), cred, nil)
-		containerClient := service.NewContainerClient(os.Getenv("AzureBlobStorageContainer"))
+		containerClient, _ := service.NewContainerClient(os.Getenv("AzureBlobStorageContainer"))
```
Comment (shubham1172) [inline tests/certification/bindings/azure/blobstorage/blobstorage_test.go:561]: Should we handle the error and fail the test?

File: tests/certification/bindings/azure/blobstorage/blobstorage_test.go [hunk 2]
```diff
@@ -558,16 +558,16 @@ func TestBlobStorage(t *testing.T) {
...
 
-		blobClient := containerClient.NewBlockBlobClient("snapshotthis.txt")
-		uploadResp, uploadErr := blobClient.UploadBufferToBlockBlob(
+		blobClient, _ := containerClient.NewBlockBlobClient("snapshotthis.txt")
```
Comment (shubham1172) [inline tests/certification/bindings/azure/blobstorage/blobstorage_test.go:563]: Same comment as above.

**Review by shubham1172 — APPROVED** [review]

File: tests/certification/bindings/azure/blobstorage/blobstorage_test.go (hunk 1 above)
Comment (berndverst) [reply tests/certification/bindings/azure/blobstorage/blobstorage_test.go:561]: Normally I'd say we should handle it but it really doesn't matter here which is why I didn't :)

**Comment by berndverst** [comment]
The swallowing of errors added is fine FYI. This is a separate SDK I'm using to directly insert and validate blob storage contents without going through the component.
If these fail the test will fail and that's all we care to know.

**Review by artursouza — APPROVED** [review]

**Pull request merged**